### PR TITLE
Rename wasm

### PR DESCRIPTION
**Pull request by guibescos**
I realized we also need to publish this package

### CODE REVIEW

File: staking/package.json [hunk 1]
```diff
@@ -1,22 +1,29 @@
 {
   "name": "@pythnetwork/staking",
-  "version": "1.3.2",
+  "version": "1.3.5",
   "description": "Pyth Network Staking SDK",
   "main": "lib/app/index.js",
   "types": "lib/app/index.d.ts",
   "license": "MIT",
+  "files": [
```
Comment (guibescos) [inline staking/package.json:8]: Drive-by fix

**Review by cctdaniel — APPROVED** [review]

**Comment by cctdaniel** [comment]
do we need to add a ci file to publish this when a release is made?

**Comment by guibescos** [comment]
We will add CI later tyvm
